### PR TITLE
WizradV2: fix recommendation packages

### PR DESCRIPTION
**Pull request by mgold1234**
this commits fix the recommendation packages when getting empty result

### CODE REVIEW

**Review by Andrewgdewar — APPROVED** [review]

File: src/Components/CreateImageWizardV2/steps/Packages/PackageRecommendations.tsx [hunk 1]
```diff
@@ -112,7 +112,7 @@ const PackageRecommendations = () => {
             {isSuccess && !data && (
               <>No recommendations found for the set of selected packages</>
             )}
-            {isSuccess && data && data[0].packages && (
+            {isSuccess && data && data[0]?.packages && (
```
Comment (lucasgarfield) [inline src/Components/CreateImageWizardV2/steps/Packages/PackageRecommendations.tsx:115]: Couldn't this just be `isSuccess && data[0].packages && ...`?

Comment (regexowl) [reply src/Components/CreateImageWizardV2/steps/Packages/PackageRecommendations.tsx:115]: Not sure if this PR is up-to-date as API schema changed in the morning (https://github.com/osbuild/image-builder/pull/1165). Fix for this bug is a part of https://github.com/osbuild/image-builder-frontend/pull/2024

Comment (Andrewgdewar) [reply src/Components/CreateImageWizardV2/steps/Packages/PackageRecommendations.tsx:115]: I have the included fix in my branch if you want to close this.

Comment (mgold1234) [reply src/Components/CreateImageWizardV2/steps/Packages/PackageRecommendations.tsx:115]: we can close this pr, because the API schema changed

Comment (mgold1234) [reply src/Components/CreateImageWizardV2/steps/Packages/PackageRecommendations.tsx:115]: and this fix appears in @Andrewgdewar's pr

**Pull request closed without merging**